Note that certain client policies are incompatible with OHTTP. (#67)
unicorn-wg/oblivious-http: draft-thomson-http-oblivious.md

encapsulated requests is integrity protected and authenticated so that it can
be attributed to the oblivious request resource; see {{key-configuration}}.

Since clients connect directly to the proxy instead of the target, application
configurations wherein clients make policy decisions about target connections,
e.g., to apply certificate pinning, are incompatible with Oblivious HTTP.  In
such cases, alternative technologies such as HTTP CONNECT

> ({{Section 9.3.6 of HTTP}}) should be used. Applications can implement related

Reviewer: ```suggestion
({{Section 9.3.6 of HTTP}}) can be used. Applications could implement related
```

> policies on key configurations and proxy connections, though these may not

Reviewer: ```suggestion
policies on key configurations and proxy connections, though these might not
```

provide the same properties as policies enforced directly on target
connections. When this difference is relevant, applications can instead connect

> directly to the target at the cost of privacy.

Reviewer: ```suggestion
directly to the target at the cost of either privacy or performance.
```

{{?FORWARDED=RFC7239}}, and any similar information.  A client does not depend
on the proxy using an authenticated and encrypted connection to the oblivious request
resource, only that information about the client not be attached to forwarded
requests.  As a consequence, application configurations wherein clients make
policy decisions about target connections, e.g., to apply certificate pinning,

> are incompatible with Oblivious HTTP.  In such cases, alternate technologies

Reviewer: ```suggestion
are incompatible with Oblivious HTTP.  In such cases, alternative technologies
```

> such as HTTP CONNECT {{?HTTP-CONNECT=RFC7231, Section 4.3.6}}.

Author: ```suggestion
such as HTTP CONNECT {{?HTTP-CONNECT=RFC7231, Section 4.3.6}} should be used.
```

> such as HTTP CONNECT {{?HTTP-CONNECT=RFC7231, Section 4.3.6}} should be used.

Reviewer: ```suggestion
such as HTTP CONNECT ({{Section 9.3.6 of HTTP}}) should be used.
```
